tf_c.sh

```diff
@@ -12,7 +12,11 @@ PROCID=$(pgrep tf_linux64 | head -n 1)
 
 if [ "$DEBUG" = true ] || [ "$COMPILE" = true ]; then
     echo "Compiling shared library with $(find . -name '*.c')"
-    gcc $(find . -name '*.c') -shared -fpic -g -o tf_c.so -Wall
+    gcc $(find . -name '*.c') -shared -fpic -g -o tf_c.so -Wall #I would suggest switching to Makefile, over this.
```

Comment:
I'm open to using a Makefile but I think at this point, since I'm not linking with any libraries or really have to do anything special during build, its better to keep it simple and remove the `make` dependency.

Reply:
That's completely fair. I only wrote that comment out of frustration with the script injecting the dl regardless of if the build process failed, and was familiar with how Makefile would stop on failure.

Though, in the same pull request, I've added a check in the script for if the build does fail. How you've implemented your automatic building is clever, and doesn't have any build speed concerns because the project is written in C :)